### DISCUSSION TITLE
raspap on docker (rasp-OS-64) Failed to allocate manager object: Read-only file system

### DISCUSSION THREAD

**Discussion by selfmade01** [RaspAP/raspap-docker#15] [General]
Hello
i try to install raspa-docker.
 i ask in this thread becuse i'm not sure raspao-docker-diskussion watch anybody..

after my first step in install 
`docker run --name raspap -it -d --privileged --network=host -v /sys/fs/cgroup:/sys/fs/cgroup:ro --cap-add SYS_ADMIN jrcichra/raspap-docker`
i get this error at the logs
`systemd 241 running in system mode. (+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN -PCRE2 default-hierarchy=hybrid)
Detected virtualization docker.
Detected architecture arm64.

Welcome to Debian GNU/Linux 10 (buster)!

Set hostname to <server>.
Failed to create /init.scope control group: Read-only file system
Failed to allocate manager object: Read-only file system
[!!!!!!] Failed to allocate manager object.
Exiting PID 1...`

**Comment by NL-TCH** [comment]
hi, check #18 and the proposed fixes in #19 and #20

TLDR: run with `docker run --name raspap -it -d --privileged --network=host --cgroupns=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cap-add SYS_ADMIN jrcichra/raspap-docker`
